### DISCUSSION TITLE
ZKSYNC Plugins

### DISCUSSION THREAD

**Discussion by alexyr255** [Cyfrin/foundry-full-course-cu#2710] [Q&A] [answered]
I can't use ZKSync plugin like before. Now it shows :
"You are using an outdated version of the zkSync plugin: v0.6.1-7, please update to v0.6.2-24 by force-refreshing the page and clearing your browser cache."

I tried to refresh the page , clear cache, reopen in the different browser but it's not working. I can compile but can't deploy.

**Comment by EngrPips** [comment, marked as answer]
Hello @alexyr255, The `zksync`part is optional, and that's because working with `zksync` isn't very smooth right now. If you face too much difficulty doing the `zksync` section of the course you can leave it and proceed.
